harness/determined/cli/command.py

```diff
@@ -224,8 +224,12 @@ def list_tasks(args: Namespace) -> None:
         if "workspaceId" in item:
             wId = item["workspaceId"]
             del item["workspaceId"]
-            assert wId in w_names, f"workspace id {wId} is not found"
-            item["workspaceName"] = w_names[wId]
+            # TODO: come up with a better fix that can convey workspace name?
+            assert wId == 0 or wId in w_names, f"workspace id {wId} is not found"
+            if wId == 0:
+                item["workspaceName"] = "all workspaces"
```

Comment:
hmm this would be the default workspace not all workspace I think. this is just setting up how the entry is rendered a single task can't belong to all workspaces